runelite-client/src/main/java/net/runelite/client/game/PlayerManager.java

```diff
@@ -211,6 +204,7 @@ public void updateStats(Player requestedPlayer)
 		});
 	}
 
+	@Subscribe
```

Comment:
these annotations are never processed on this class so might as well not be here